### PR TITLE
PUBDEV-7405: Add "single_node_mode" for building CoxPH model

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

File: h2o-algos/src/main/java/hex/coxph/CoxPHModel.java [hunk 1]
```diff
@@ -51,6 +51,8 @@
 
     public boolean _calc_cumhaz = true; // support survfit
 
+    public boolean _single_node_mode = false;
```
Comment (honzasterba) [inline h2o-algos/src/main/java/hex/coxph/CoxPHModel.java]: seem unused

File: h2o-algos/src/test/java/hex/coxph/CoxPHTest.java [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
+import java.util.HashSet;
+import java.util.Set;
 
 import static org.junit.Assert.*;
 
 public class CoxPHTest extends TestUtil {
 
   @BeforeClass
-  public static void setup() { stall_till_cloudsize(1); }
+  public static void setup() { stall_till_cloudsize(2); }
```
Comment (honzasterba) [inline h2o-algos/src/test/java/hex/coxph/CoxPHTest.java]: intentional?

File: h2o-algos/src/main/java/hex/coxph/CoxPHModel.java (hunk 1 above)
Comment (michalkurka) [reply h2o-algos/src/main/java/hex/coxph/CoxPHModel.java]: it is used all over the place :)

File: h2o-algos/src/test/java/hex/coxph/CoxPHTest.java (hunk 2 above)
Comment (michalkurka) [reply h2o-algos/src/test/java/hex/coxph/CoxPHTest.java]: no! thanks - this is also causing the tests to fail

File: h2o-algos/src/main/java/hex/coxph/CoxPHModel.java (hunk 1 above)
Comment (honzasterba) [reply h2o-algos/src/main/java/hex/coxph/CoxPHModel.java]: on sorry, i thought this was declared on model class, I was only looking the file name and did not realize this is on params

**Pull request merged**